spec/music_album_spec.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'helper_spec'
+
+describe MusicAlbum do
+  before :each do
+    @music_album = MusicAlbum.new(true, '2017-01-01')
+    @music_album.genre = Genre.new(1, 'Gospel')
+  end
+  context 'given two arguments' do
```

Comment:
Please make a different file for testing genre class. As it should be more clear and concise ways to write test cases